third_party/move/testing-infra/transactional-test-runner/src/vm_test_harness.rs

```diff
@@ -448,8 +449,14 @@ static MOVE_STDLIB_COMPILED: Lazy<Vec<CompiledModule>> = Lazy::new(|| {
 #[derive(Debug, Clone, PartialOrd, Ord, PartialEq, Eq)]
 pub enum TestRunConfig {
     CompilerV1,
-    CompilerV2 { v2_experiments: Vec<(String, bool)> },
-    ComparisonV1V2 { v2_experiments: Vec<(String, bool)> },
+    CompilerV2 {
```

Comment:
I think this could be restructured to explicitly involve compiler version enums (and allow for comparisons between various combinations), which allow for future compiler versions as well.

But maybe this is a follow up task and not for this PR.

Reply:
Good idea, but wasn't really the objective of this PR.